pom.xml

```diff
@@ -2777,7 +2777,7 @@
           <dependency>
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
-            <version>8.14</version>
+            <version>8.23</version>
```

Comment:
Can you upgrade `project/plugins.sbt`'s checkstyle version too?

Reply:
Done!